client/my-sites/subscribers/components/subscriber-data-views/subscriber-data-views.tsx

```diff
@@ -0,0 +1,201 @@
+import { useBreakpoint } from '@automattic/viewport-react';
+import { DataViews } from '@wordpress/dataviews';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import TimeSince from 'calypso/components/time-since';
+import { EmptyListView } from 'calypso/my-sites/subscribers/components/empty-list-view';
+import { SubscriberLaunchpad } from 'calypso/my-sites/subscribers/components/subscriber-launchpad';
+import { SubscriberProfile } from 'calypso/my-sites/subscribers/components/subscriber-profile';
+import { useSubscribersPage } from 'calypso/my-sites/subscribers/components/subscribers-page/subscribers-page-context';
+import { useSubscriptionPlans } from 'calypso/my-sites/subscribers/hooks';
+import { Subscriber } from 'calypso/my-sites/subscribers/types';
+import { useSelector } from 'calypso/state';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import { isSimpleSite } from 'calypso/state/sites/selectors';
+import { SubscribersSortBy } from '../../constants';
+import type { View, Field, Action } from '@wordpress/dataviews';
+import './style.scss';
+
+const SubscriptionTypeCell = ( { subscriber }: { subscriber: Subscriber } ) => {
+	const plans = useSubscriptionPlans( subscriber );
```

Comment:
I found that we're not returning plans data from the endpoint used here. So this is how it works in the individual subscriber page, but it's not actually working yet.